examples/main.tf

```diff
@@ -0,0 +1,17 @@
+data "aws_vpcs" "default" {
```

Comment:
there should be a subfolder below examples to stay in sync with our other modules. We usually have something like examples/basic/main.tf for example